client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -242,27 +246,30 @@ function HelpModal({
           </form>
         ) : (
           <>
-            <p>
-              <Trans i18nKey='learn.tried-rsa'>
-                <a href={RSA} rel='noopener noreferrer' target='_blank'>
-                  placeholder
-                </a>
-              </Trans>
-            </p>
-            <div className='alert alert-danger'>
+            <div className='alert'>
               <FontAwesomeIcon icon={faExclamationCircle} />
```

Comment:
Should we remove the icon too? I feel like without the warning callout this just feels out of place.

Reply:
Sure. I will add that to the change-set.